src/Plane.js

```diff
@@ -1,26 +1,48 @@
+const Airport = require(`./Airport`);
+
 class Plane { 
     flightNumber;
     takenOff;
     takeOff;
     notInThisAirport;
+    inThisAirport;
     land;
+    noTakeOffRequired;
 
     constructor(flightNumber, takenOff = true) {
         this.flightNumber = flightNumber;
         this.takenOff = takenOff;
+       
     }
 
     notInAirport() {
         if (this.takenOff == true) {
             this.notInThisAirport = true;
         }
 
-        if (this.notInThisAirport == true) {
-            this.takeOff = false;
-            this.land = true;
-        }
     }
+    // inAirport(){
+    //     if (this.inThisAirport == false) {
+    //         this.takeOff = false;
+    //         this.land = true;
+    //     } 
 
+    //     if (this.inThisAirport == true) {
+    //         this.takeOff = true;
+    //         this.land = false;
+    //     } 
+
+    // }
+    
+    noTakeOff() {
+        this.flightNumber = `PG123`;
```

Comment:
I defined the variable here to make the test work. Maybe should have created an instance and ran the test on that.